app/actions/actionsTypes.ts

```diff
@@ -28,7 +28,9 @@ export const ROOM = createRequestTypes('ROOM', [
 	'DELETE',
 	'REMOVED',
 	'FORWARD',
-	'USER_TYPING'
+	'USER_TYPING',
+	'HISTORY_REQUEST',
+	'HISTORY_FINISHED'
```

Comment:
This naming is anti-pattern, but I don't see any value adding success/failure atm